Updated es_es.json

muriplz opened the pull request.

paladini-maker committed:
Updated es_es.json

paladini-maker committed:
Updated es_es.json

paladini-maker committed:
Update es_es.json

paladini-maker committed:
Revised and improved translation of the ponder section

paladini-maker committed:
Updated ponder section

More fixes.

paladini-maker committed:
Updated ponder section

More fixes...

paladini-maker committed:
Updated es_es.json

Many arrangements
General improvements to the ponder section

After a couple of hours of work it looks like this is up to date. =)
@muriplz You can check from here and test the template, I finished checking it in game and it looks good.

paladini-maker committed:
Update es_es,json

Ready for review

paladini-maker committed:
Update es_es.json

Fixed some items such as cinder flour, now called Nether flour, because it is made with Nether rock and is used to create blaze cakes.

muriplz committed:
Update es_es.json

-Reloj de cuco is accepted by RAE, "Cuco" is the name of the bird
-elemento/item >> Objeto
-All caps in the middle of the sentence that I see will be changed to non- caps
- the "signos de puntuación" exclamation and question mark, didnt make sense on some cases
-Googles >> gafas del ingeniero (they used to be called like that, but just "Gafas" would fit that much)
-Shift is not the same as Mayús izq and can be missleading
-Click is not an anglicism , its an onomatopeia, the spanish onomatopeia is "Clic"
- minor fixes

muriplz commented:
ive been literally 10 hours trying to pull request to your fork @FrannDzs , lots of forks from people appeared but not yours, only way i had to was this one or making a pull request that wasnt 100% updated with your lasts commits, sorry. I had to start all over, so expect more updates here, or tell me what else can i do so you can merge /review it easier

muriplz commented:
Sticker is "Pegatina", maybe the translation could be "Pegatina" or "Bloque de pegatina" , doesnt look good to be honest, but i cant think of anything better

FrannDzs commented:
> ive been literally 10 hours trying to pull request to your fork @FrannDzs , lots of forks from people appeared but not yours, only way i had to was this one or making a pull request that wasnt 100% updated with your lasts commits, sorry. I had to start all over, so expect more updates here, or tell me what else can i do so you can merge /review it easier

Your changes are fine, I had already lowered the capital letters but the rest is fine, let's follow that criterion.

About the shift key:
https://es.wikipedia.org/wiki/May%C3%BAs_(tecla)

It's not a whim, and it's well said, it's just that I translated Immersive Engineering with that criteria and if we change I would have to completely revise it again....

Sticker>pegatina are ok, reloj de cuco accepted. Click>clic accepted

muriplz commented:
Yeah, mayús izq.  Is correct, my bad

FrannDzs commented:
> Yeah, mayús izq. Is correct, my bad

Ok, make the changes, did you try using github desktop? it's better for working with this...

paladini-maker committed:
Merge branch 'Creators-of-Create:mc1.16/dev' into mc1.16/dev

muriplz commented:
Yeah i use it

muriplz committed:
Mayús izq.

muriplz committed:
Pegatina

muriplz committed:
Pegamento

there was "super pegamento", "súper pegamento" and "pegamento" , so all of them are now just "pegamento"

muriplz committed:
Merge branch 'FrannDzs:mc1.16/dev' into mc1.16/dev

muriplz committed:
Mayúz izdo. like minecraft has it
